website/docs/usage/processing-pipelines.md

```diff
@@ -362,6 +362,18 @@ nlp = spacy.load("en_core_web_sm", disable=["tagger", "parser"])
 nlp.enable_pipe("tagger")
 ```
 
+In addition to `disable`, `spacy.load()` also accepts `enable`. If `enable` is
+set, all components except for those in `enable` are disabled.
+
+```python
+# Load the complete pipeline, but disable all components except the tagger
+nlp = spacy.load("en_core_web_sm", enable=["tagger"])
+# Has the same effect, as the parser is already not part of enabled set of components
+nlp = spacy.load("en_core_web_sm", enable=["tagger"], disable=["parser"])
+# Will raise an error, as the sets of enabled and disabled components are conflicting
+nlp = spacy.load("en_core_web_sm", enable=["tagger"], disable=["tagger"])
```

Comment:
These examples are going to be confusing with the `tok2vec` situation, but maybe it would better to use it so it's clear. Maybe use the pairs `["tok2vec", "tagger"]` + `["ner"]` for the example so it's one with `tok2vec` and one without?

Reply:
Not sure what you mean. Like this?
```python
nlp = spacy.load("en_core_web_sm", enable=["tok2vec", "tagger"])
nlp = spacy.load("en_core_web_sm", enable=["tok2vec", "tagger"], disable=["ner"])
nlp = spacy.load("en_core_web_sm", enable=["tok2vec", "tagger"], disable=["tagger"])
```

Reply:
Yes, or `["ner"]` / `["ner"]` if that's clearer in the last example.